### DISCUSSION TITLE
Audio and Image files content

### DISCUSSION THREAD

**Discussion by manjirisulekavadi** [Sunbird-Ed/Community#574] [Feature request]
Digital Green has a requirement to have content type as audio file and image file so that on the consumption side, audio files and image files can be fetched like the videos and pdfs. Currently the only work around is to embed audio files or images as a common content type through the content creation and not through upload as files.  
@Krishnaj20

**Comment by Krishnaj20** [comment]
@manjirisulekavadi - The backend content service APIs allow for creation of new content type with mime type as audio and image. However, the content upload editor will need some changes to enable this capability. We will be happy to help you to extend Sunbird to build and contribute this change. 

@pallakartheekreddy @vrayulu @rajeevsathish @vinukumar-vs

**Reply by manjirisulekavadi** [reply]
Thanks @Krishnaj20 
@manojlondhe
